docs/stacks-guide-azure.md

```diff
@@ -415,15 +426,22 @@ Stacks Guide document][stacks-guide-continue] so we can pick up where we left
 off.
 
 <!-- Links -->
-[crossplane-cli]: https://github.com/crossplaneio/crossplane-cli
-[crossplane-azure-networking-docs]: TODO
+[sample-wordpress-stack]: https://github.com/crossplaneio/sample-stack-wordpress
+
+[crossplane-cli]: https://github.com/crossplaneio/crossplane-cli/tree/release-0.1
+[crossplane-azure-networking-docs]: https://github.com/crossplaneio/crossplane/blob/master/design/one-pager-resource-connectivity-mvp.md#microsoft-azure
 [stacks-guide]: stacks-guide.html
 [provider-azure-guide]: cloud-providers/azure/azure-provider.md
 
-[crossplane-concepts]: TODO
-[portable-claims]: TODO
+[stack-docs]: https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-stacks.md#crossplane-stacks
+[stack-azure]: https://github.com/crossplaneio/stack-azure
+
+[crossplane-concepts]: concepts.md
+[portable-classes-docs]: https://github.com/crossplaneio/crossplane/blob/master/design/one-pager-default-resource-class.md
 
 [azure]: https://azure.microsoft.com
 [azure-vnet-rule]: https://docs.microsoft.com/en-us/azure/mysql/concepts-data-access-and-security-vnet
+[azure-resource-group-docs]: https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview
 
 [stacks-guide-continue]: stacks-guide.html#install-support-for-our-application-into-crossplane
```

Comment:
This should be `stacks-guide.md#install-support-for-our-application-into-crossplane`